share/completions/wesnoth.fish

```diff
@@ -28,7 +28,7 @@ complete -c wesnoth -l exit-at-end --description 'Exits once the scenario is ove
 #complete -c wesnoth -l controller<number> --description 'Selects the controller for this side'
 complete -c wesnoth -l era --description 'Use this option to play in the selected era instead of the "Default" era'
 complete -c wesnoth -l nogui --description 'Runs the game without the GUI'
-complete -c wesnoth -l parm<number> --description 'Sets additional parameters for this side'
```

Comment:
I'm pretty sure we should rename that parameter - I'm guessing this was supposed to be a poor man's hint. Still unfortunate fish_indent breaks (assuming the completion works at all).